vars/signArtifacts.groovy

```diff
@@ -15,15 +15,16 @@ SignArtifacts signs the given artifacts and saves the signature in the same dire
 @param Map[platform] <Required> - The distribution platform for signing.
 */
 void call(Map args = [:]) {
-
     if (args.sigtype.equals('.rpm')) {
-        echo "RPM Add Sign"
+        withCredentials([string(credentialsId: 'jenkins-rpm-signing-props', variable: 'configs')]) {
```

Comment:
Have you run test signing for rpm package with these changes? We should be careful with this change as this is built in our distribution workflow.

Reply:
I have updated the credentials and the regression files seems fine to me.

Reply:
Tested! Ran successfully